### ISSUE TITLE
Unable to create new users

### ISSUE THREAD

**Issue by aptonline** [jhalter/mobius#19]
On testing the brew install server version 0.3.0 I found that I couldn't save a new user.  I receive the following error in terminal and the hotline task list in client shows `saving list of accounts...` with no progress bar and then the server times out.

```
2022-06-04T11:14:33.200+0100	error	Unimplemented transaction type received	{"UserName": "apt", "RequestID": 349}
2022-06-04T11:16:49.275+0100	error	error serving request	{"RemoteAddr": "192.168.1.247:49193", "err": "read tcp 192.168.1.125:5500->192.168.1.247:49193: read: operation timed out"}
```

I'm using Hotline Client 1.9 on Mac OS 9.

**Comment by aptonline** [comment]
Ok so I think I found the issue, I see that Mobius doesn't support 1.9 client.  Reverting to Hotline 1.2.3 fixed the issue for me and it saves the users as expected now.

**Comment by jhalter** [comment]
The 1.9 client should be fully supported except that I haven't implemented the transaction type used by the multi-account editor yet: #2 

It should work if you use the single account editor instead:

![Untitled](https://user-images.githubusercontent.com/868228/172013209-d1e7c144-ae1e-4ea3-9246-9f3e1df98719.jpg)

**Comment by jhalter** [comment]
Hi, this is implemented in v0.4.0 now, available through brew, docker etc.
